guillotina/fields/annotation.py

```diff
@@ -133,43 +146,19 @@ def __init__(self, *args, value_type=None, bucket_len=5000, **kwargs):
         await operation(obj, anno_context, value['value'])
 
 
-@configure.value_deserializer(IBucketListField)
-def field_converter(field, value, context):
-    if not isinstance(value, dict):
-        raise ValueDeserializationError(field, value, 'Not an object')
-    operation_name = value.get('op', 'undefined')
-    operation = query_adapter(field, IPatchFieldOperation, name=operation_name)
-    if operation is None:
-        raise ValueDeserializationError(
-            field, value, f'"{operation_name}" not a valid operation')
-    if 'value' not in value:
-        raise ValueDeserializationError(
-            field, value, f'Mising value')
-    return value
-
-
-@configure.value_serializer(BucketListValue)
-def dynamic_list_converter(value):
-    if value is None:
-        return
-    return {
-        'len': len(value),
-        'buckets': len(value.annotations_metadata)
-    }
-
-
 @configure.adapter(
     for_=IBucketListField,
     provides=IPatchFieldOperation,
     name='append')
 class PatchBucketListAppend(patch.PatchListAppend):
+    value_factory: typing.Any = BucketListValue
```

Comment:
Why its needed ?

Reply:
Because it's generic for the dict field implementation